velox/common/base/StatsReporter.h

```diff
@@ -53,10 +53,10 @@
 namespace facebook::velox {
 
 enum class StatType {
-  AVG,
-  SUM,
-  RATE,
-  COUNT,
+  kAvg,
```

Comment:
It would be nice to document these. It is not clear what kRate is, for example.

Reply:
Commented per Meta internal definitions of these counters. OSS implementation shall follow the same semantics.